jsk_recognition_utils/node_scripts/polygon_array_publisher.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright: Yuki Furuta <furushchev@jsk.imi.i.u-tokyo.ac.jp>
```

Comment:
But I could not find other nodes that have license header.
Even if this is preferred, I think we should create another PR to update it for all these nodes.